.github/CONTRIBUTION_GUIDELINES.md

```diff
@@ -0,0 +1,33 @@
+# Contributing to Umbraco CMS
+
+When you’re considering creating a pull request for Umbraco CMS, we will categorize them in two different sizes, small and large.
+
+The process for both sizes is very similar, as [explained in the contribution document](CONTRIBUTING.md#how-do-i-begin).
+
+## Small PRs
+Bug fixes and small improvements - can be recognized by seeing a small number of changes and possibly a small number of new files.
+
+We’re usually able to handle small PRs pretty quickly. A community volunteer will do the initial review and flag it for Umbraco HQ as “community tested”. If everything looks good, it will be merged pretty quickly [as per the described process](REVIEW_PROCESS.md).
+
+### Up for grabs
+
+Umbraco HQ will regularly mark newly created issues on the issue tracker with the `Up for grabs` tag. This means that the proposed changes are wanted in Umbraco but the HQ does not have the time to make them at this time. These issues are usually small enough to fit in the "Small PRs" category and we encourage anyone to pick them up and help out.  
```

Comment:
I don't believe the **Up for grabs** label is exclusive to smaller issues. Larger issues may of course be more complex and take more time to implement, but could still fit under the **Up for grabs** label.

It may however be worth mentioning that a dialog with either Umbraco HQ or the PR team is more important for as the size and complexity of the issues grow.

Reply:
I have kept this plain and simple by saying "It is highly recommended that you speak to the HQ before making large, complex changes. "
because its already covered https://github.com/umbraco/Umbraco-CMS/blob/v8/dev/.github/CONTRIBUTING.md#guidelines-for-contributions-we-welcome

Reply:
This text is fine for now, we may mark larger things as up for grabs too but then we will also describe how to implement them before we do so.